content/blog/2018-05-09-onboarding-social-weather.md

```diff
@@ -0,0 +1,672 @@
+| seem more unified and consistent.                                                                                                                                                                                                                                                                                                                                 |   1.126978|
+| very much appreciated!                                                                                                                                                                                                                                                                                                                                            |   1.125833|
+| \- well organized, readable code                                                                                                                                                                                                                                                                                                                                  |   1.100000|
+| \- wow very extensive testing! well done, very thorough                                                                                                                                                                                                                                                                                                           |   1.100000|
+| \- i’m delighted that you find my work interesting and i’m very keen to help, contribute and collaborate in any capacity.                                                                                                                                                                                                                                         |   1.084493|
+| thank you very much for your thorough and thoughtful review, @batpigandme ! this is great feedback, and i think that visdat will be much improved because of these reviews.                                                                                                                                                                                       |   1.083653|
+| great, thank you very much for accepting this package. i am very grateful about the reviews, which were very helpful to improve this package!                                                                                                                                                                                                                     |   1.074281|
+
+As you can imagine, these sentences make the whole team very happy! And
+we hope they’ll encourage you to contribute to our mission to make
+scientific software better.
```

Comment:
> our mission to make scientific software better.

this sounds a bit too simplistic when referring to our mission https://ropensci.org/about/

Reply:
oh again bad phrasing, will remove.